### ISSUE TITLE
Outputs object is always empty

### ISSUE THREAD

**Issue by followbl** [amondnet/now-deployment#25]
Is there anything I'm doing wrong here? I'm having trouble accessing the output from the now-deployment step...found adding the id to the step in the forums, but still no luck. Any assistance if I'm missing something simple is appreciated.

```
name: CI
on: [pull_request]
jobs:
  deploy:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v1

      - name: Deploy
        uses: amondnet/now-deployment@v1
        with:
          zeit-token: ${{ secrets.ZEIT_TOKEN }} # Required
          zeit-team-id: team_123 #Optional
          now-args: "--scope team123 --confirm"
        id: build_zeit

      - name: Cypress Test
        uses: cypress-io/github-action@v1
        with:
          config: baseUrl=${{ steps.build_zeit.outputs.preview-url }}
```

**Comment by peterschrott** [comment]
Also ran into that issue.

Its only set if comment on commit or pull is enabled.

**Comment by followbl** [comment]
@peterschrott I owe you a ☕, works beautifully. @amondnet will leave this open for you as it seems you add the bug label (and the bug without the PR comment enabled could still exist)...but for now @peterschrott solved the issue at hand. Thanks all 🥇

**Comment by amondnet** [comment]
@followbl 
fixed. Could you try with v2.0.1?

**Comment by followbl** [comment]
@amondnet tested this right when you ping'ed me and then forgot to comment. unfortunately for me the output is empty still if I take away `github-token`. Adding that keeps the output from being null.

**Comment by hakonkrogh** [comment]
@amondnet Looks like the issue with getting the preview url can be solved quite easily since the preview url is the only thing stored in the `myOutput` variable in the deployment step. I've forked your repo and I'm using my fork successfully in my projects:

https://github.com/hakonkrogh/now-deployment

I can create a pull request for this if you want to.

**Comment by amondnet** [comment]
@hakonkrogh Yes, I would appreciate it.

**Comment by hakonkrogh** [comment]
> @hakonkrogh Yes, I would appreciate it.

https://github.com/amondnet/now-deployment/pull/38

**Comment by amondnet** [comment]
@followbl  closed this in #38
